### PR TITLE
Documentation for the Resources features + draft release notes

### PR DESCRIPTION
**Pull request by carohadad**

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -15,6 +15,25 @@ To receive monthly updates on new features and improvements to Tella and stay up
 ## Tella - Latest Versions {#tella---latest-versions}
 
 
+### Android: Tella 2.7.0 (?) - Released on March X, 2024 {#android-tella-270-?---released-on-march-X-2024}
+- Added support for a content distribution feature that allow organizations to securely disseminate resources and materials from a Tella Web space to their on-the-ground users directly within Tella’s encrypted container. This includes creating PDFs on Tella Web and sharing training material, guides for data collection, or documents. Resources can be accesed offline after download. More information [here](/tella-web#managing-resources).
+- Technical maintanance tasks:
+  - Bugfixing
+  - Refactoed the Tella Web project's screen to use the new navigation activity.
```
Comment (djordjemarusic) [inline docs/whats-new.md]: "Refactored" is misspelled.  Also, refactoring of the navigation is in the background, it refers to the mechanism of how to go from one screen to another, there are no changes on the screen and maybe it's best just to say "The navigation within the Tella Web feature has been refactored for better stability and scalability."

File: docs/For organizations/tella-web.md [hunk 2]
```diff
@@ -156,6 +165,40 @@ After selecting the users to add to the project, click the "Add" button to save
...
+users directly within Tella’s encrypted container. Resources can be used to submit instructions for data collection, messages or guides on how to collect data. 
+
+#### Creating resources to the space {#creating-resources-to-the-space}
+
+Admins can create resources to the space by navigating to the **Resources** left hand sidebar. For the moment resources can be created by uploading PDFs. We recommend keeping those smaller than 20MB to make sure users can download those even if internet is slow.
+
+Resources are shared within the space. Keep in mind that resources are only available to users after adding those to specific projects. 
+
+On the same screen, space admins can manage resources. The following options are available
+- Dowload resources individually or dowloading multiple resources.
```
Comment (djordjemarusic) [inline docs/For organizations/tella-web.md]: The "Download" is misspelled as "Dowload" few times, maybe it's best to put the whole text to some spell checker.

File: docs/For organizations/tella-web.md [hunk 3]
```diff
@@ -9,23 +9,26 @@ slug: /tella-web
...
 
-Tella Web is a great option to collect "reports" from users. Reports can include photos, videos, audio recordings, pdf documents and text.
+Tella Web is a great option to collect "reports" from users. Reports can include photos, videos, audio recordings, pdf documents and text. Organizations can also upload PDFs to each Tella Web project and users can access these resources directly from their Tella vault.
 
 Setting up Tella Web involves three steps, which are thoroughly discussed in this section:
 
 * **Install Tella Web**: This step entails installing the Tella Web server and configuring the domain name to ensure its proper functioning.
-* **Set up your project(s) on your server**: In this step, admins decide how to organize themselves to collect data, create “projects”, and set roles and permissions for their users.
+* **Set up your project(s) on your server**: In this step, admins decide how to organize themselves to collect data, create "projects", add "resources" for users to download on their devices and and set roles and permissions for users.
```
Comment (djordjemarusic) [inline docs/For organizations/tella-web.md]: There is "and and" instead of "and".

File: docs/For organizations/tella-web.md (hunk 2 above)
Comment (carohadad) [reply docs/For organizations/tella-web.md]: solved!

**Pull request merged**